### PR TITLE
Bumping wasmtime 1.0.1 -> 2.0.2

### PR DESCRIPTION
**Pull request by dwdking**
Bumping wasmtime 1.0.1 -> 2.0.2

### CODE REVIEW

File: crates/deterministic-wasi-ctx/Cargo.toml [hunk 1]
```diff
@@ -11,12 +11,13 @@ categories = ["wasm"]
 [dependencies]
 anyhow = "1.0.56"
 async-trait = "0.1.53"
-cap-std = "0.25.2"
-wasi-common = ">= 0.34.0, < 1.1.0"
+cap-std = "0.26.1"
+cap-primitives = "0.26.1"
+wasi-common = ">= 0.34.0, < 2.1.0"
```
[the remark this answers is not in the document]
Comment (dwdking) [reply crates/deterministic-wasi-ctx/Cargo.toml]: Correct, the changes don't work on wasi-common < 2.0.0. I've modified the line to be `wasi-common = ">= 2.0.0, < 2.1.0"'

File: crates/deterministic-wasi-ctx/Cargo.toml [hunk 2]
```diff
@@ -11,12 +11,13 @@ categories = ["wasm"]
 [dependencies]
 anyhow = "1.0.56"
 async-trait = "0.1.53"
-cap-std = "0.25.2"
-wasi-common = ">= 0.34.0, < 1.1.0"
+cap-std = "0.26.1"
```
Comment (mkcny) [inline crates/deterministic-wasi-ctx/Cargo.toml:14]: It looks like this isn't even used anymore?

File: crates/deterministic-wasi-ctx/src/clocks.rs [hunk 3]
```diff
@@ -1,4 +1,6 @@
-use cap_std::time::{Duration, Instant, SystemTime};
+use std::time::Duration;
```
Comment (mkcny) [inline crates/deterministic-wasi-ctx/src/clocks.rs:1]: Why not https://docs.rs/cap-primitives/latest/cap_primitives/time/struct.Duration.html ?

Comment (dwdking) [reply crates/deterministic-wasi-ctx/src/clocks.rs]: @jeffcharles, I moved to use this but I think it's just exported from the rust lib.

File: crates/deterministic-wasi-ctx/Cargo.toml (hunk 2 above)
Comment (dwdking) [reply crates/deterministic-wasi-ctx/Cargo.toml]: Correct, removed from the dependencies.

**Review by mkcny — APPROVED** [review]

**Pull request merged**